composer.json

```diff
@@ -3,6 +3,7 @@
     "description": "Lets you find a path to the system CA bundle, and includes a fallback to the Mozilla CA bundle.",
     "type": "library",
     "license": "MIT",
+    "minimum-stability": "dev",
```

Comment:
👎

Reply:
Why has this been added?

Reply:
Sorry, I actually thought it was necessary for the next developments for the Sf5 compatibility, but asked to a colleague, it isn't. I originally added it to install Sf5 on my local machine and run the tests.
Fixing that :)

Reply:
It's necessary to test locally with Symfony 5 (which hasn't been released yet, but is about to be), but this must be removed from the PR.